valhalla/midgard/point2.h

```diff
@@ -194,7 +194,7 @@ template <typename PrecisionT> class PointXY : public std::pair<PrecisionT, Prec
     return false;
   }
 
-protected:
+  virtual ~PointXY() = default;
```

Comment:
nit: can you move this up to the top of the class? it feel strange for it to be at the bottom

Reply:
sure